dev/wlp-jakartaee-transform/rules/jakarta-xml-servletContainerInitializer.properties

```diff
@@ -0,0 +1 @@
+javax.servlet.ServletContainerInitializer=jakarta.servlet.ServletContainerInitializer
```

Comment:
Since this file has the same content as `jakarta-cdiServletInitializer.properties`, do we need two files? Ought to have a better naming strategy for these files too. Perhaps this is a change for a follow on PR?

Reply:
Yeah, it has the same content and we don't need both. When you added CDI I didn't realize you added the cdi one. Do you mind if I remove one in Jakarta authorization PR?

Reply:
Took this down as an action item and will do this in the authorization PR.